lightly/openapi_generated/swagger_client/models/jobs_data.py

```diff
@@ -0,0 +1,258 @@
+# coding: utf-8
+
+"""
+    Lightly API
+
+    Lightly.ai enables you to do self-supervised learning in an easy and intuitive way. The lightly.ai OpenAPI spec defines how one can interact with our REST API to unleash the full potential of lightly.ai  # noqa: E501
+
+    OpenAPI spec version: 1.0.0
+    Contact: support@lightly.ai
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+import pprint
+import re  # noqa: F401
+
+import six
```

Comment:
and this one too?

Reply:
Yes, the additional dependencies from the generated files are already in the branch develop_active_learning, see https://github.com/lightly-ai/lightly/blob/90741ff6e723633c64eeba069c31dc54229c55ee/requirements/base.txt#L9. All the dependencies are quite small.